contracts/UFragmentsPolicy.sol

```diff
@@ -57,38 +56,48 @@ contract UFragmentsPolicy is Ownable {
     // The number of rebase cycles since inception
     uint256 public epoch;
 
-    uint256 private constant RATE_DECIMALS = 18;
-
-    uint256 private constant TARGET_RATE = 1 * 10**RATE_DECIMALS;
-
-    int256 private constant TARGET_RATE_SIGNED = int256(TARGET_RATE);
+    uint256 private constant DECIMALS = 18;
 
     // Due to the expression in computeSupplyDelta(), MAX_RATE * MAX_SUPPLY must fit into an int256.
     // Both are 18 decimals fixed point numbers.
-    uint256 private constant MAX_RATE = 10**6 * 10**RATE_DECIMALS;
+    uint256 private constant MAX_RATE = 10**6 * 10**DECIMALS;
     // MAX_SUPPLY = MAX_INT256 / MAX_RATE
     uint256 private constant MAX_SUPPLY = ~(uint256(1) << 255) / MAX_RATE;
 
+    // CPI-U value AUG 1983 100.2, DECIMALS Fixed point number
+    uint256 private constant BASE_CPI = 1002 * 10**(DECIMALS-1);
```

Comment:
Seems like we should explain why this is DECIMALS -1.

Reply:
I've added an example. Is that sufficient?

Reply:
Kind of an issue. It's 99.9 on July and 100.2 in August. 
https://www.usinflationcalculator.com/inflation/consumer-price-index-and-annual-percent-changes-from-1913-to-2008/

Reply:
Assuming it's a linear increase. 99.9 on July 1st and 100.2 on August 1st. We can choose July 10th as when it was 100.0 ..